packages/edit-navigation/src/components/toolbar/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { Spinner, SelectControl, Popover } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { BlockToolbar } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import SaveButton from './save-button';
+import BlockInspectorDropdown from './block-inspector-dropdown';
+import AddMenuForm from './add-menu-form';
+
+export default function Toolbar( {
+	menus,
+	selectedMenuId,
+	isAddingMenu,
+	onSelectMenu,
+	onCancelAddingMenu,
+} ) {
+	const hasSelectedMenu = !! menus && !! selectedMenuId;
+
+	return (
+		<div className="edit-navigation-toolbar">
+			{ hasSelectedMenu || isAddingMenu ? (
+				<>
+					{ hasSelectedMenu && (
+						<SelectControl
+							className="edit-navigation-toolbar__menu-select"
+							label={ __( 'Currently editing' ) }
+							hideLabelFromVision
+							value={ selectedMenuId }
+							options={ menus.map( ( menu ) => ( {
+								label: menu.name,
+								value: menu.id,
+							} ) ) }
+							onChange={ onSelectMenu }
+						/>
+					) }
+
+					{ ! isAddingMenu && (
+						<>
+							<BlockToolbar __experimentalExpandedControl />
```

Comment:
I think `BlockToolbar` should be wrapped in a `NavigableToolbar` for arrow key navigation to work like in the post editor.

Reply:
Done in 654e1a1!